setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = arrayfire
-version = 3.7.20200213
+version = 3.8.0
```

Comment:
Please separate the version change to `3.8` into separate commit and correct the date of `3.7.*` to relfect tomorrow's release for 3.7 release.

Reply:
I just noticed that we can't use current master branch's HEAD for v3.7 release as `af_bitnot` is in the middle of all the fixes and features that can go into v3.7. Note: bitnot made available in v3.8